src/fastga/models/handling_qualities/tail_sizing/update_ht_area.py

```diff
@@ -16,21 +16,23 @@
 import openmdao.api as om
 
 from scipy.constants import g
+from stdatm import Atmosphere
```

Comment:
I don't see why you moved that import here. I fanything it should be after typing since this package use the typing package. Following that logic it should be before FAST-OAD if a move is really required ...